### PR TITLE
Added StarDix and GladMC

### PR DESCRIPTION
**Pull request by JobsonMarinho**
# Type of change
- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update
# Checklist
- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com/)
- [x] I have created the manifest.json with the required values
# Further comments

### CODE REVIEW

File: minecraft_servers/gladmc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "gladmc",
+  "nice_name": "GladMC",
+  "direct_ip": "jogar.gladmc.com",
+  "server_wildcards": [
+    "%.gladmc.com"
+  ],
+  "social": {
+    "web": "https://gladmc.com",
+    "web_shop": "https://loja.gladmc.com",
```
Comment (JakobBruening) [inline minecraft_servers/gladmc/manifest.json:10]: 404

File: minecraft_servers/gladmc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      "command": "/play fp"
+    },
+    "bedwars": {
+      "name": "BedWars",
+      "command": "/play bw"
+    }
+  },
+  "discord": {
+    "server_id": 978540914970619934,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/gladmc/manifest.json:33]: Only supported for partner servers atm.

File: minecraft_servers/stardix/manifest.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      "command": "/play md"
+    },
+    "survival":{
+      "name": "Survival",
+      "command": "/play sv"
+    }
+  },
+  "discord": {
+    "server_id": 447833437697474562,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/stardix/manifest.json:41]: Only supported for partner servers atm.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/gladmc/manifest.json (hunk 1 above)
Comment (JobsonMarinho) [reply minecraft_servers/gladmc/manifest.json]: Fixed

File: minecraft_servers/gladmc/manifest.json (hunk 2 above)
Comment (JobsonMarinho) [reply minecraft_servers/gladmc/manifest.json]: Fixed

File: minecraft_servers/stardix/manifest.json (hunk 3 above)
Comment (JobsonMarinho) [reply minecraft_servers/stardix/manifest.json]: Fixed

**Pull request merged**